src/vow.re

```diff
@@ -120,10 +181,37 @@ module Result: ResultType = {
     |> Vow.flatMapUnhandled(x => return(x))
     |> onError(() => fail(handler()));
   let unwrap = (transform, vow) => Vow.flatMap(transform, vow);
+  let all2 = ((v1, v2)) =>
```

Comment:
I might be overlooking something but I think that since this is using `Result.flatMap` it's handling the `Js.Result.Ok`/`Js.Result.Error` which you don't get with the `Vow.all2` method.